### PR TITLE
Feature/fda

### PR DESCRIPTION
**Pull request by MarcPfeif**
Just OpenFDA feature  - Removed Billing File

### CODE REVIEW

File: app/medispan/views.py [hunk 1]
```diff
@@ -34,3 +35,58 @@ def find_gpi(request):
     data['drugName'] = drugName
 
     return JsonResponse(data)
+
+
+def openFDA(request):
+    if request.method == 'GET':
+        if Drug.objects.filter(ndc_upc_hri = request.GET['ndc']).exists():
+            ndc = openfda_ndc(request.GET['ndc'])
+            URL = "https://api.fda.gov/drug/label.json?api_key=WLm7AKMaBm8CYrA1pKJrcbmw8z36kFjv7XcqH3uJ&search=openfda.product_ndc:"+ ndc
```
Comment (waltertschwe) [inline app/medispan/views.py]: can we get this API key into an environment variable (.env)

File: app/medispan/views.py [hunk 2]
```diff
@@ -34,3 +35,58 @@ def find_gpi(request):
...
+            URL = "https://api.fda.gov/drug/label.json?api_key=WLm7AKMaBm8CYrA1pKJrcbmw8z36kFjv7XcqH3uJ&search=openfda.product_ndc:"+ ndc_string + "&limit=1"
+            data = requests.get(URL).json()
+
+            openfda[drug_name] = data
+
+        OpenFDA.objects.create(ndc=ndc_string,drug_name = drug_name, drug_data =openfda)
+        return JsonResponse(openfda)
+    return JsonResponse(["Error: 403","Invalid GPI"], safe =False)
+
+def openfda_ndc(ndc):
```
Comment (waltertschwe) [inline app/medispan/views.py]: just add a comment above this function what it does and why you need to get this value into openFDA form.  We know now, but in 3 months we will look at this and have no idea why we are doing it.

**Comment by MarcPfeif** [comment]
Ok I've made the changes

**Comment by waltertschwe** [comment]
LGTM

**Pull request merged**